capstone/src/main/webapp/profile-loader.js

```diff
@@ -33,7 +33,7 @@ function getStudentInfo() {
     // Add additional student information
     var personalInfo = document.getElementById('student-info');
     const newLine = '<br>';
-    personalInfo.innerHTML = 'Email: ' + studentInfo['email'] + newLine;
+    personalInfo.innerHTML += 'Email: ' + studentInfo['email'] + newLine;
```

Comment:
Wanted to double check that you intend to do += here and not just =? Making sure you didn't need to reset this innerHTML or something.

Reply:
Hm, I think either way it should reset it since the tag with ID 'student-info' is initially empty, but I will change it back to '='  to ensure that it resets. Thanks for pointing this out!